AK/JsonValue.cpp

```diff
@@ -89,6 +89,46 @@ JsonValue& JsonValue::operator=(JsonValue&& other)
     return *this;
 }
 
+bool JsonValue::equals(const JsonValue& other) const
+{
+    if (is_null() && other.is_null())
+        return true;
+
+    if (is_bool() && other.is_bool() && as_bool() == other.as_bool())
+        return true;
+
+    if (is_string() && other.is_string() && as_string() == other.as_string())
+        return true;
+
+#if !defined(KERNEL) && !defined(BOOTSTRAPPER)
+    if (is_number() && other.is_number() && to_number<double>() == other.to_number<double>()) {
+        return true;
+    }
+#else
+    if (is_number() && other.is_number() && to_number<i64>() == other.to_number<i64>()) {
+        return true;
+    }
```

Comment:
Why is the 64-bit underlying storage not enabled outside of the kernel?

Reply:
For my understanding, the to_number<double> will do the trick. All numbers could be compared when the value is converted to double first. In terms of computing power maybe not the best option... Shall I add cases for each number type individually? :-)

Reply:
Ah I geddit. Nah, that's fine :)